tests/installation/upgrade_select.pm

```diff
@@ -33,10 +33,11 @@ sub run {
     # hardware detection and waiting for updates from suse.com can take a while
     assert_screen_with_soft_timeout('select-for-update', timeout => 500, soft_timeout => 100, bugref => 'bsc#1028774');
     send_key $cmd{next};
-    assert_screen [qw(remove-repository license-agreement)], 240;
-    if (match_has_tag("license-agreement")) {
+    assert_screen [qw(remove-repository license-agreement license-agreement-accepted)], 240;
+    if (match_has_tag("license-agreement") || match_has_tag("license-agreement-accepted")) {
```

Comment:
I have a little concern about this part, the soft_failure bsc#1077703 would be recored with any version product here. 
eg : http://openqa-apac1.suse.de/tests/252#step/upgrade_select/2,  the license version is SLES15, but it will cause soft failure anyway.

Reply:
@Jiawei517 Yes, the version is SLES 15, but the license should NOT be shown here.
There are two issues recorded in bsc#1077703: 
1, incorrect license agreement version (fixed now)
2, incorrect time to show the license agreement, please refer to:
https://bugzilla.suse.com/show_bug.cgi?id=1077703#c8

Reply:
thanks for explanation, MR merged.